src/main/java/core/basesyntax/service/impl/ConvertWriteDataServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.service.ConvertWriteDataService;
+import java.util.Map;
+
+public class ConvertWriteDataServiceImpl implements ConvertWriteDataService {
+    private static final String TITLE_STRING = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+    private Map<String, Integer> fruitStorageState;
+
+    @Override
+    public String getReport() {
+        fruitStorageState = new FruitDaoImpl().getStorageState();
```

Comment:
create a field FruitDao fruitDao; and pass an instance of the dao through the constructor